website/docs/table-group/data-table/data-table-code.md

```diff
@@ -176,35 +174,37 @@ If fixed columns aren't visible in the following example, try reducing the windo
 
 :::
 
-## Multi-level header
+### Columns merging
```

Comment:
this I'd actually move to "cells" and rename to "merging cells horizontally", because "columns merging" sounds like we merge entire columns together

Reply:
Visually it looks like we are merging cells horizontally, but from the code side we are merging columns. I would leave the original title so as not to confuse developers when they are looking for a case of how to merge columns (but they will need to look for it in the section on cells).

Hmm, maybe it's better to update the example, so it wouldn't be confusing visually.

Reply:
Discussed with Ilia, both "Columns merging" and "Columns grouping" are good. "Meging cells horizontally/vertically" don't suit for these cases.